requirements.txt

```diff
@@ -12,3 +12,4 @@ accelerate>=0.25.0
 scipy>=1.10.0
 bitsandbytes>=0.42.0
 nltk>=3.8.1
+luxonis-ml[all]
```

Comment:
We could pin this to `luxonis-ml[all]>=0.1.0`

Reply:
Done